tests/player_helpers.cpp

```diff
@@ -72,6 +72,8 @@ void clear_character( player &dummy )
     // This sets HP to max, clears addictions and morale,
     // and sets hunger, thirst, fatigue and such to zero
     dummy.environmental_revert_effect();
```

Comment:
Anyone have any idea whether this originally intended to reset calories along with hunger "and such"?